react/components/Dropdown/index.js

```diff
@@ -159,19 +162,24 @@ ${this.getDropdownIdentification()}`
       containerClasses += 'ba b--disabled bw1 bg-disabled '
     } else if (error || errorMessage) {
       containerClasses += 'ba b--danger hover-b--danger '
+    } else if (inline) {
+      containerClasses += 'fw5 '
     } else {
       containerClasses += 'bg-base hover-b--muted-3 ba b--muted-4 '
     }
 
     return (
-      <div className="vtex-dropdown">
+      <div className={`vtex-dropdown ${inline ? 'dib' : ''}`}>
```

Comment:
Colocar uma nova classe quando for inline

```js
className={`vtex-dropdown ${inline ? 'vtex-dropdown--inline dib' : ''}`}
```

Reply:
Atualizado.